test/output/callbacks/record.json

```diff
@@ -41,7 +41,7 @@
       "station_api_bind_address": "0.0.0.0", 
       "station_api_port": 8888, 
       "station_discovery_string": "OPENHTF_DISCOVERY", 
-      "station_id": "jethier.mtv.corp.google.com", 
+      "station_id": "fahhem.mtv.corp.google.com", 
```

Comment:
we should probably not be exporting google's internal network names to an open github repo.  i don't think it realllly matters, but can we anonymize this string?